README.md

```diff
@@ -0,0 +1,25 @@
+This repo is home for the Single Page Application (SPA) project templates which ship as part of ASP.NET Core and the .NET SDK.
+At the moment, those are `Angular` and `React` templates.
+
+Besides these two templates, we also want this repo to serve as a knowledge-base for all community-owned SPA project templates out there.
+We expect to maintain that list in the `SpaProjectTempltes.md` file. If you know of any such project templates, which will meet some basic quality bar, submit a PR and we'll consider it. Things that we'll be looking for are:
```

Comment:
Instead of maintaining the community template list in a separate file, can we just list them in the readme? One less click.